docker-compose.yml

```diff
@@ -22,8 +21,6 @@ services:
       - SAVE_MEMORY_SNAPSHOTS=${SAVE_MEMORY_SNAPSHOTS:-false}
     ports:
       - ${CORE_PORT:-1865}:80
-    volumes:
-      - ./core:/app
```

Comment:
This volume is necessary here, as the code is obtained via git. Outside of this repo, it make more sense to use the built image (already containing the cat code) and opening a volume just on plugins

Reply:
As above